setup.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, *args, **kwargs):
         'cached_property>=1.0',
         'functools32;python_version<"3.2"',
         'h5py>=2.2,<3',
-        'libsonata>=0.1.4',
+        'libsonata<=0.1.4;python_version<"3.6"',
```

Comment:
I hope we will drop this soon enough. 
libsonata api did not change and the result are identical. Python2 users will just not benefit from the perf improvement